.goreleaser.yml

```diff
@@ -16,24 +23,29 @@ builds:
     goarch:
       - 386
       - amd64
-      - arm
       - arm64
     ignore:
       - goos: darwin
         goarch: 386
+      - goos: linux
+        goarch: 386
 archives:
   -
-    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    name_template: '{{ .ProjectName }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
     replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
```

Comment:
Why is Linux And Windows removed?

Reply:
This is replacement section for name. Go uses term Darwin for macOS, but we refer as macos, hence I used it . If we don’t include any other term then default will be used, in this case windows/linux